db/migrate/20230815193556_add_ref_users_to_posts.rb

```diff
@@ -0,0 +1,6 @@
+class AddRefUsersToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :author, null: false, foreign_key: true
+    add_index :posts, :author
```

Comment:
- [ ] The `foreign_key` attribute should refer to the users, even if the column id would be `author_id`

  ```
  ...foreign_key: { to_table: :users }
  ```

  ![image](https://github.com/Luiscarlosvd/blog-app-rails/assets/81468675/0396d6dd-3ee4-4a99-9784-7d8c5aaeca44)

- [ ] Kindly remove this `add_index` line as `add_reference` above already adds the index.